backend/opencv_backend.py

```diff
@@ -0,0 +1,52 @@
+import cv2
+
+from backend import Backend, PublishFunc
+from canvas import Canvas
+
+class OpenCVBackend(Backend):
+    """ Gets and displays video using the OpenCV (`cv2`) library.
+
+        This is intended for use with laptops with webcams, not on the
+        Raspberry Pi. """
+
+    def __init__(self, width: int, height: int, publish_recording_status_func: PublishFunc):
+        super().__init__(width, height, publish_recording_status_func)
+
+        self.webcam = None
+
+        framerate = 60
+        # Time between video frames when running on OpenCV, in milliseconds
+        self.frametime = 1000 / framerate
```

Comment:
Cast this as `int(1000 / framerate)` since `cv2.waitKey` requires an integer not a float.